main.c

```diff
@@ -1,46 +1,75 @@
 /*
-This program is free software: you can redistribute it and/or modify
+This file is part of TicTacToe.
+
+TicTacToe is free software: you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 (at your option) any later version.
 
-This program is distributed in the hope that it will be useful,
+TicTacToe is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
-along with this program.  If not, see <https://www.gnu.org/licenses/>.
+along with TicTacToe.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-
-
 #include <stdio.h>
 #if !defined(NXDK)
 #include <SDL2/SDL.h>
 #else
 #include <SDL.h>
 #define printf(...) debugPrint(__VA_ARGS__)
 #endif
+#include "include/graphics.h"
+
+int width;
+int height;
+SDL_Event event;
 
 void Init() {
```

Comment:
One might think about moving the graphical-init into the new `graphics.c` you created. So that we end up with this in `main.c` i.e.:
```C
void Init() {
    init_gamedate();
    window = init_graphics();
    ....
}

Reply:
Sounds good, I'll do that